src/app/components/Article/index.jsx

```diff
@@ -1,20 +1,34 @@
 import React, { Fragment, Component } from 'react';
 import Helmet from 'react-helmet';
 import 'isomorphic-fetch';
-import styled from 'styled-components';
 import Header from '../Header';
-
-import {C_EBON, FF_NEWS_SANS_REG} from '../../../lib/constants/styles';
-
-const Headline = styled.h1`
-  color: ${C_EBON};
-  font-family: ${FF_NEWS_SANS_REG};
-  font-size: 2em;
-`;
+import MainContent from '../MainContent';
 
 class Article extends Component {
   state = {
-    headline: 'Article Headline',
+    data: {
+      blocks: [
```

Comment:
I think this data structure should match the data from optimo, certainly from the root model: https://github.com/bbc/simorgh/blob/latest/data/prod/scenario-27.json#L21. So: `data.model.blocks`.

Furthermore, we should use the `blocks` helper that is [here](https://github.com/bbc/simorgh/blob/latest/src/app/models/blocks.js), rather than defining these blocks manually.